src/styles/components/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  font-family: ' Roboto ', sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: $main-text-color;
+  box-sizing: border-box;
+  transition-duration: 0.3s;
+
+  &__image {
+    width: 100%;
+    height: 134px;
+    object-fit: contain;
+    margin-bottom: 36px;
+  }
+
+  &__main {
+    margin-bottom: 16px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &_text {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &_number {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__action {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    border: 1px solid $color-action;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: $main-bg-color;
+    background-color: $color-action;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $color-action;
+      background-color: $main-bg-color;
+    }
+  }
+
+  &:hover {
+    & .card__title {
+      color: #34568b;
+    }
+
+    transform: scale(120%);
```

Comment:
The transform property in the hover state should use a decimal for scaling instead of a percentage. Instead of 'scale(120%)', it should be written as 'scale(1.2)'. Using a percentage is not a valid value for the scale function.